imx/examples/workflows/prepareWithdrawal.go

```diff
@@ -1,106 +1,99 @@
-package withdrawals
+package main
 
 import (
 	"context"
 	"encoding/json"
 	"log"
 
-	"github.com/immutable/imx-core-sdk-golang/examples/workflows/utils"
-	"github.com/immutable/imx-core-sdk-golang/generated/api"
-	converter "github.com/immutable/imx-core-sdk-golang/internal/convert"
-	"github.com/immutable/imx-core-sdk-golang/signers"
-	"github.com/immutable/imx-core-sdk-golang/tokens"
-	withdrawalsWorkflow "github.com/immutable/imx-core-sdk-golang/workflows/withdrawals"
+	"github.com/immutable/imx-core-sdk-golang/imx"
+	"github.com/immutable/imx-core-sdk-golang/imx/api"
+	"github.com/immutable/imx-core-sdk-golang/imx/internal/convert"
```

Comment:
I think it would be better to remove the internal imports as SDK users will not be able to import from internal. We don't need the denomination conversion to demonstrate the example.

Reply:
But it provides the clarity of what a user need to do, otherwise a user will have to go through the same journey as we did when we were testing this.

Reply:
I think both issues raised here are worth resolving.

Reply:
This will be addressed separately in subsequent PRs